include/lsst/ctrl/events/Event.h

```diff
@@ -79,8 +74,12 @@ class Event
     static const std::string UNINITIALIZED;
 
     Event();
+    Event(const PropertySet& ps);
```

Comment:
Rather than naming the variable after the type (`ps`), choose something more descriptive, about what it represents.